app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,67 @@
+package com.codesoom.assignment;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+
+/**
+ * 자바 HTTP 서버를 만들어보기 위한 데모 핸들러입니다.
```

Comment:
학습용 프로젝트이긴 하지만, 프로의 기분으로 주석을 작성해보는 연습을 해보시는 것이 좋을 것 같습니다. JavaDoc 주석은 짧고 간결하며, 코드의 내용을 설명하되 코드의 내용이 수정되어도 가급적 주석이 바뀌지 않도록 작성하는 것이 좋습니다.

```suggestion
 * HTTP 핸들러.
```

주석을 읽어보면 각종 http 메소드와 uri의 기능이 모두 설명되어 있는데요, 기능이 바뀌면 클래스 주석도 변경될 것이므로 약간 냄새가 난다고 할 수 있습니다.

앞으로 작업하시면서 각 http mothod-uri 별로 메소드가 분리되면 주석도 함께 분리될 거에요.